packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_controller_v2.dart

```diff
@@ -456,25 +390,12 @@ class LoggingControllerV2 extends DisposableController
   }
 
   void log(LogDataV2 log) {
-    List<LogDataV2> currentLogs = List.of(data);
-
-    // Insert the new item and clamped the list to kMaxLogItemsLength.
-    currentLogs.add(log);
-
-    // Note: We need to drop rows from the start because we want to drop old
-    // rows but because that's expensive, we only do it periodically (eg. when
-    // the list is 500 rows more).
-    if (currentLogs.length > kMaxLogItemsUpperBoundV2) {
-      int itemsToRemove = currentLogs.length - kMaxLogItemsLowerBoundV2;
-      // Ensure we remove an even number of rows to keep the alternating
-      // background in-sync.
-      if (itemsToRemove % 2 == 1) {
-        itemsToRemove--;
-      }
-      currentLogs = currentLogs.sublist(itemsToRemove);
-    }
-
-    _updateData(currentLogs);
+    data.add(log);
+    // TODO(@CoderDake): Add filtersearch behavior
+    // TODO(@CoderDake): Add Search behavior
+    // TODO(@CoderDake): Add Selection update behavior
+    // TODO(@CoderDake): Add status update
```

Comment:
nit: format todos with TODO(issue) or TODO(g3-username) otherwise this will hit a lint in the g3 roll